version.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs')

```

Comment:
Prettier should have appended a `;` here.

Reply:
Prettier seems to ignore JS files in the root folder.